### PR TITLE
[4.0] [plg_users_terms] Error when creating user

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for https://github.com/joomla/joomla-cms/pull/26341#issuecomment-532117701  .

### Summary of Changes

Fixes an error when creating user with  `User - Terms and Conditions` plugin enabled.

### Testing Instructions

Enable `User - Terms and Conditions` plugin.
Create a new user.

### Expected result

Users created without errors.

### Actual result

User created but with error:
>Call to a member function addLog() on bool 

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by SharkyKZ** [comment]
@infograf768 please test.

File: layouts/plugins/user/terms/label.php [hunk 1]
```diff
@@ -82,34 +82,35 @@
...
+		'data-toggle' => 'modal',
+		'data-target' => '#tosModal',
+	];
+
 	$link = HTMLHelper::_('link', Route::_($article->link . '&tmpl=component'), $text, $attribs);
+
+	echo HTMLHelper::_(
+		'bootstrap.renderModal',
+		'tosModal',
+		array(
```
Comment (Quy) [inline layouts/plugins/user/terms/label.php]:
```suggestion
		[
```

File: layouts/plugins/user/terms/label.php [hunk 2]
```diff
@@ -82,34 +82,35 @@
...
+			'url'    => Route::_($article->link . '&tmpl=component'),
+			'title'  => $text,
+			'height' => '100%',
+			'width'  => '100%',
+			'modalWidth'  => '800',
+			'bodyHeight'  => '500',
+			'footer' => '<button type="button" class="btn btn-secondary" data-dismiss="modal" aria-hidden="true">'
+				. Text::_("JLIB_HTML_BEHAVIOR_CLOSE") . '</button>',
+		)
```
Comment (Quy) [inline layouts/plugins/user/terms/label.php]:
```suggestion
		]
```

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on ca218c698c7ce7c2cd0b07abfb3e3d5ebad4906e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26343">issues.joomla.org/tracker/joomla-cms/26343</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**